apps/articles/client/editorial_signup.coffee

```diff
@@ -4,6 +4,7 @@ Backbone = require 'backbone'
 editorialSignupLushTemplate = -> require('../templates/editorial_signup_lush.jade') arguments...
 Cycle = require '../../../components/cycle/index.coffee'
 { resize } = require '../../../components/resizer/index.coffee'
+analyticsHooks = require '../../../lib/analytics_hooks.coffee'
```

Comment:
Looks like this isn't used in this view, is this a mistype or something that is going to be addressed later?